### PR TITLE
add JSON import/export of linearRangeActionRaoResults + fix sonar

### PR DESCRIPTION
**Pull request by baptsegu**
Signed-off-by: Baptiste Seguinot <baptiste.seguinot@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [X] The commit message follows our guidelines
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Remove cost from LinearRangeActionResult as it cannot be used by the searchTreeRao if it is not directly included in the RaoComputationResult
Add json import/export of LinearRangeActionResult

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

no

### CODE REVIEW

**Comment by baptsegu** [comment]
@terriervik, I'm requested your re-reviewing as I added a fix for sonar since your last review. The JSON import/export part have not change since your last approval, only the travis.yml and the main pom.

@murgeyseb Sonar was apparently not computing the test coverage since yesterday, I had to change a few things in `travis.yml` and add a profile to fix it. I'm not against a double check on that part.

File: pom.xml [hunk 1]
```diff
@@ -294,6 +294,32 @@
...
             </distributionManagement>
         </profile>
+        <profile>
+            <id>coverage</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.jacoco</groupId>
+                        <artifactId>jacoco-maven-plugin</artifactId>
+                        <version>0.8.5</version>
```
Comment (murgeyseb) [inline pom.xml]: Perhaps a property for JaCoCo version would be better

Comment (baptsegu) [reply pom.xml]: property is now used

**Comment by baptsegu** [comment]
Modification brought to version property

**Pull request merged**